### ISSUE TITLE
Load Savegame i get "The given key was not present in the dictionary"

### ISSUE THREAD

**Issue by Ursch** [voxeltycoon/bug-tracker#178]
When i load a Savegame from yesterday i got "The given key wos not present in the dictionary".

**Comment by Ursch** [comment]
![image](https://user-images.githubusercontent.com/70261139/91280870-aef34080-e787-11ea-965c-2cf559b06782.png)

**Comment by andrewpey** [comment]
Is the problem still reproduce? Can we have the [log file](https://github.com/voxeltycoon/bug-tracker/blob/master/README.md#logs)?

**Comment by SMERKIN5000** [comment]
I came across this issue and I think I know the reason why. I clicked on a train and selected to move it to a depot and accidentally clicked a garage window instead of a train depot window. The game worked OK at that point however after that they train was always greyed out when I looked at a station. I could never move it back to a depot. When I clicked the link to view the 1st person I saw other trains pass through it and third person it wasn't visible. I saved the game and exited to desktop and then tried to load the game again to see if the train would fix itself. This is when I receive the message reported by this thread. Save file can now no longer be loaded.

[Player.log](https://github.com/voxeltycoon/bug-tracker/files/6368824/Player.log)
